client/my-sites/plan-features/index.jsx

```diff
@@ -524,6 +532,7 @@ export default connect(
 				const newPlan = isNew( plan ) && ! isPaid;
 				const currentPlan = sitePlan && sitePlan.product_slug;
 
+				const isRelatedToCurrentPlan = planObject ? planConstantObj.relatedPlan === sitePlan.product_slug : false;
```

Comment:
yeah, it started being a selector... but at the end, the relevant info is not in the redux subtree, but in the plans constants file. So since we were already getting the plan id from the selector, it felt like adding a non-necessary layer to add a selector that were not really selecting anything from the redux tree